Android/config/quality/checkstyle_config.xml

```diff
@@ -4,7 +4,6 @@
     "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
 
 <module name="Checker">
-
```

Comment:
why this change?

Reply:
there is nothing change just remove a simple space.

Reply:
then, remove this file from commit

Reply:
@AminullahTajMuhammad if you are not making any relevant change to some file, you shouldn't add it to your git staging and commits since it adds you to the file's contributor list making it hard to track in future who worked on what

Reply:
remove this file from commit